src/main/java/org/killbill/billing/client/model/SubscriptionUsage.java

```diff
@@ -0,0 +1,76 @@
+package org.killbill.billing.client.model;
+
+import java.util.List;
+import java.util.UUID;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class SubscriptionUsage  {
```

Comment:
Should we call it `SubscriptionUsageRecord` for consistency with `SubscriptionUsageRecordJson`?